source/elements/oneMKL/source/architecture/api_design.inc.rst

```diff
@@ -34,18 +35,18 @@ oneMKL uses C++ STL data types for scalars where applicable:
 * Integer scalars are C++ fixed-size integer types (``std::intN_t``, ``std::uintN_t``).
 * Complex numbers are represented by C++ ``std::complex`` types.
 
-In general, scalar integer arguments to oneMKL routines are 64 bit integers (``std::int64_t`` or ``std::uint64_t``). Integer vectors and matrices may have varying bit widths, defined on a per-routine basis.
+In general, scalar integer arguments to oneMKL routines are 64-bit integers (``std::int64_t`` or ``std::uint64_t``). Integer vectors and matrices may have varying bit widths, defined on a per-routine basis.
 
 .. _onemkl_dpcpp_datatypes:
 
 DPC++ datatype usage
 ++++++++++++++++++++
 
-oneMKL uses following SYCL Language data types and DPC++ Language Extensions data types:
+oneMKL uses following SYCL language data types and DPC++ language extensions data types:
```

Comment:
SYCL is not a language and DPC++ does not have language extensions. A feature that is an extension today may be a standard feature tomorrow. Distinguishing between SYCL and DPC++ seems pointless, unless you are committed to supporting a SYCL subset. Keep it simle:

oneMKL uses the following  DPC++  data types:

Reply:
Updated